proxies/live/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml

```diff
@@ -2,12 +2,12 @@
     async="false"
     continueOnError="false"
     enabled="true"
-    name="KeyValueMapOperations.GetSecureVariables"
+    name="KeyValueMapOperations.GetSecureVariablesNHSLogin"
```

Comment:
I'm not sure we should be changing this policy as it is also used elsewhere i.e. in StatusEndpoint flow. I think a new version for nhs login like you've done for CIS2 would be best

Reply:
e2c2510